### ISSUE TITLE
亲爱的作者，在代码中J = order.size(dim=1)导致后面arr = y_pred[:, K_d:].reshape(J, K_d)维度不对，后续的symbol = torch.cat((symbol[:, 0:int(K_d/2)], torch.zeros(1, l*K_d), symbol[:, int(K_d/2):K_d]), dim=-1)中 torch.zeros(1, l*K_d),维度不匹配。我将Reshape去掉，torch.zeros(1, l*K_d)中1修改为64，得到的损失为Epoch: 0, Batch: 0, Loss: inf。请查看代码，感谢您分享的代码。期待你的回复。

### ISSUE THREAD

**Issue by ManXu1** [NickMts/Energy-Efficient-OFDM-with-Intelligent-PAPR-aware-Adaptive-Modulation#1]

**Comment by xiaobaijh** [comment]
还不如不开源，过分了
